app/templates/view_teams.html

```diff
@@ -29,72 +29,63 @@ <h3>Team Details</h3>
     Team Email Address : {{team.emailAddress}}
   </div>
 
-  <div>
-    <h3> Team Lead and Members' Permissions</h3>
-    {% call(item) summary.list_table(
-            team.teamLeads,
-            field_headings=[                    
-            'Email Address',
-            'Name',
-            'Team Lead',
-            'Download Reports',
-            'Publish Opporunities',
-            'Create Drafts',
-            'Download Responses',
-            'Answer Seller Questions', 
-            'Create Work Orders'
-            ],
-            field_headings_visible=True)
-        %}
-            {% call summary.row() %}
-            {% for i in team.teamLeads %}
-              {% for key, value in team.teamLeads[i].items() %}
-                {% if key == 'emailAddress' %}
-                  {% call summary.field() %}
-                    <a href="{{ url_for('.find_user_by_email_address', email_address = value)}}" method="get"  class="question">{{ value}}</a>    
-                  {% endcall %}
-                {% else %}
-                  {{ summary.field_name(value) }}
-                  <!-- {% call summary.field() %} -->  
-                  {% endcall %}
-                {% endif %}
-              {% endfor %}
-              <!-- Team lead's permissions will always be true-->
-                {% for i in range(7) %}
-                  {{ summary.field_name(True) }}
-                {% endfor %}
-            {% endfor %}
-            {% endcall %}
+<div> 
+  <br> 
+  <h3> Team Leads Table</h3>
+  {% call(item) summary.list_table(
+    team_leads,
+    field_headings=[
+      'Name',                    
+      'Email Address',
+    ],
+    field_headings_visible=True)
+%}
+  {% call summary.row() %}
+    {{ summary.text(item.name) }}
+    {% call summary.field() %}
+      <a href="{{ url_for('.find_user_by_email_address', email_address = item.emailAddress)}}" method="get"  class="question">{{ item.emailAddress}}</a>    
```

Comment:
```suggestion
      <a href="{{ url_for('.find_user_by_email_address', email_address = item.emailAddress)}}" class="question">{{ item.emailAddress}}</a>    
```

I don't think you need `method="get"` here.

Reply:
removed get